### DISCUSSION TITLE
why does this transition and duration class is not working properly ?

### DISCUSSION THREAD

**Discussion by jaimeenkoffeekodes** [tailwindlabs/tailwindcss#13541] [Help]
<div className={twMerge(`grid gap-3 mt-2 md:grid-cols-2 transition-all duration-700`, !billEdit && 'hidden')}>

**Comment by wongjn** [comment]
Could you provide a [Tailwind Play](https://play.tailwindcss.com) or at least some code that you are having trouble with? Could you also explain how it is not working properly please? What do you expect to happen vs what is actually happening?
